docs/data/charts/sankey/SankeyOverview.js

```diff
@@ -0,0 +1,122 @@
+import * as React from 'react';
+import { Unstable_SankeyChart as SankeyChart } from '@mui/x-charts-pro/SankeyChart';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+
+const data = {
+  nodes: [
+    // Sub-segment
+    { id: 'Server Products', color: '#595b63' },
+    { id: 'Enterprise Services', color: '#595b63' },
+    { id: 'M365 Commercial', color: '#595b63' },
+    { id: 'M365 Consumer', color: '#595b63' },
+    { id: 'LinkedIn', color: '#595b63' },
+    { id: 'Dynamics', color: '#595b63' },
+    { id: 'Gaming', color: '#595b63' },
+    { id: 'Windows & Devices', color: '#595b63' },
+    { id: 'Search & News Ads', color: '#595b63' },
+    // Main revenue segments
+    { id: 'Intelligent Cloud', label: 'Cloud', color: '#595b63' },
+    { id: 'Productivity', color: '#595b63' },
+    { id: 'Personal Computing', label: 'Personal Computing', color: '#595b63' },
+    { id: 'Revenue', label: 'Total Revenue', color: '#595b63' },
+    // Gross Profit and breakdown
+    { id: 'Gross Profit', label: 'Gross Profit', color: '#44CE8D' },
+    { id: 'Operating Profit', label: 'Operating Profit', color: '#44CE8D' },
+    { id: 'Net Profit', label: 'Net Profit', color: '#44CE8D' },
+    { id: 'Tax', label: 'Income Tax', color: '#F35865' },
+    { id: 'Other Loss', label: 'Other Loss', color: '#F35865' },
+    { id: 'Operating Expenses', label: 'Operating Expenses', color: '#F35865' },
+    { id: 'R&D', label: 'R&D', color: '#F35865' },
```

Comment:
R&D is more well-known, but should we show the full name? At least in the tooltip.